debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -252,4 +279,14 @@ private static int validateCollectionBlacklist(Configuration config, Field field
         }
         return 0;
     }
+    
+    private static int validateDatabaseBlacklist(Configuration config, Field field, ValidationOutput problems) {
```

Comment:
It seems we have an interesting inconsistency here - MySQL connector supports adding both blacklist and whitelist but PostgreSQL requires setting of only one of those options. I'd vote going MySQL way and retrofit the functionality into Postgres connector too. Let Gunnar decide.